deployment/network-operator/templates/upgrade-crd-hook.yaml

```diff
@@ -81,15 +81,14 @@ spec:
           image: "{{ .Values.operator.repository }}/{{ .Values.operator.image }}:{{ .Values.operator.tag | default .Chart.AppVersion }}"
           imagePullPolicy: IfNotPresent
           command:
-            - /bin/sh
-            - -c
-            - >
-              kubectl apply -f /crds/network-operator;
+            - /apply-crds
+          args:
+              - --crds-dir=/crds/network-operator
         {{- if .Values.sriovNetworkOperator.enabled }}
-              kubectl apply -f /crds/sriov-network-operator;
+              - --crds-dir=/crds/sriov-network-operator
         {{- end }}
         {{- if .Values.nfd.enabled }}
-              kubectl apply -f /crds/node-feature-discovery;
+              - --crds-dir=/crds/node-feature-discovery
```

Comment:
we are missing nic-configuration-operator-chart/crds
Though not related to your PR

Reply:
If we enable everything it will also apply everything. `crds-dir` is a list and can be called multiple times

Reply:
see https://github.com/Mellanox/network-operator/pull/1118#discussion_r1824144369